cal/cal/static/js/services.js

```diff
@@ -278,6 +288,13 @@ analyticsApp.service('QueryService', function() {
       strokeWidth: 3,
     });
 
+    ctrlDetails.push({
+      values: movingAverage,
+      key: 'Moving Average Line',
```

Comment:
You should call it "XD Moving Average", where X is the number of days that you're averaging over. I actually think we should do a 30D moving average since the 5D one is still too sharp because we don't have enough data points. The moving average should average over a large enough period so that the line only makes small movements at each data point. Otherwise we might confuse the user.